lib/hammer_cli_import/configfile.rb

```diff
@@ -372,7 +387,7 @@ def build_and_upload
                                [data['org_id'].to_i, data['channel_id'].to_i])
 
           # Find the built-module .tar.gz
-          built_module_path = File.join(File.join(module_dir, 'pkg'),
+          built_module_path = File.join('pkg',
```

Comment:
for my understanding, the `chdir` on line 165 makes it ok to remove the `module_dir` here?